tutorials/core-concepts/users-and-permissions/oauth-passport.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "OpenAuth: Google, Twitter, and Facebook Login"
+layout: tutorial
+---
+
+# OpenAuth with Apostrophe Passport
+
+Apostrophe uses Passport to provide single sign-on capabilities. The `apostrophe-passport` module is a "universal bridge" compatible with pretty much all Passport strategies that follow best practices, and allows you to implement login via popular services like Google, Twitter, Facebook, Github, and Gitlab, as well as standards like OpenAuth and SAML. To learn more about Passport in general, see the official site for the [Passport strategy module](http://passportjs.org/). 
+
+## Using Apostrophe Passport
+
+Apostrophe Passport is not included with the standard Apostrophe distribution and can be installed via npm. For complete details on how to install and configure Passport, see the [official documentation on npmjs,org](https://npmjs.org/package/apostrophe-passport).
```

Comment:
npmjs.org

Reply:
Apostrophe Passport, not just Passport (because just Passport is a real thing it interfaces with and already provides, and we should not confuse them into thinking they have to install it).

Reply:
Updated.